native/acter/src/api/stream.rs

```diff
@@ -942,22 +115,106 @@ impl TimelineStream {
                 if !member.can_send_message(MessageLikeEventType::RoomMessage) {
                     bail!("No permission to send message in this room");
                 }
-                timeline.send_attachment(uri, mime_type, config).await?;
+                match *draft {
+                    MsgContentDraft::TextPlain { body } => {
+                        let content = RoomMessageEventContent::text_plain(body);
+                        timeline.send(content.into()).await;
+                    }
+                    MsgContentDraft::TextMarkdown { body } => {
+                        let content = RoomMessageEventContent::text_markdown(body);
+                        timeline.send(content.into()).await;
+                    }
+                    MsgContentDraft::Image { body, source, info } => {
```

Comment:
what happens to `body`? I don't see it being used...?

Reply:
Yeah, `body` is not used for stream send.
But it is used for `NewsEntryDraft` and `AttachmentDraft`.
As we know, news and attachment is based on media message.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1175/commits/d19f8cc4609bc80a191fcfc7e52a58035f83c76f